tests/wpt-harness/expectations/FileAPI/blob/Blob-stream.any.js.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Blob.stream()": {
+    "status": "PASS"
+  },
+  "Blob.stream() empty Blob": {
+    "status": "PASS"
+  },
+  "Blob.stream() non-unicode input": {
+    "status": "PASS"
+  },
+  "Blob.stream() garbage collection of blob shouldn't break stream consumption": {
+    "status": "PASS"
+  },
+  "Blob.stream() garbage collection of stream shouldn't break stream consumption": {
+    "status": "PASS"
+  },
+  "Reading Blob.stream() with BYOB reader": {
+    "status": "FAIL"
```

Comment:
Out of interest, what is the issue here?

Reply:
Oh, bad timing on my part. I just asked the question regarding this failure :)

Reply:
I'd have to refresh my BYOB knowledge, but it seems like that error is when this check fails:

```
if (!unwrappedStream->controller()->is<js::ReadableByteStreamController>()) {
```

That is, I think the issue is that we don't support byob streams for native stream sources?

Reply:
Added issue https://github.com/bytecodealliance/StarlingMonkey/issues/176.